From 3384ca7cf0a272d8b1cd95c7a9eb9484cdbf2edc Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Sat, 8 Jan 2011 14:31:37 -0600
Subject: [PATCH] default to fifo mode 5, for old musb beagles

The fifo mode 4, 16kb can cause corruption on musb/omap35xx,
use mode 5, 8kb...

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 drivers/usb/musb/musb_core.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/usb/musb/musb_core.c b/drivers/usb/musb/musb_core.c
index f10ff00..ee605c9 100644
--- a/drivers/usb/musb/musb_core.c
+++ b/drivers/usb/musb/musb_core.c
@@ -1060,7 +1060,7 @@ static void musb_shutdown(struct platform_device *pdev)
  */
 #if defined(CONFIG_USB_MUSB_TUSB6010) || defined(CONFIG_USB_MUSB_OMAP2PLUS) \
 	|| defined(CONFIG_USB_MUSB_AM35X)
-static ushort __initdata fifo_mode = 4;
+static ushort __initdata fifo_mode = 5;
 #elif defined(CONFIG_USB_MUSB_UX500)
 static ushort __initdata fifo_mode = 5;
 #else
-- 
1.7.5.4

